components/usage/pkg/db/workspace_instance_usage.go

```diff
@@ -49,12 +49,28 @@ func CreateUsageRecords(ctx context.Context, conn *gorm.DB, records []WorkspaceI
 	return db.CreateInBatches(records, 1000).Error
 }
 
-func ListUsage(ctx context.Context, conn *gorm.DB, attributionId AttributionID, from, to time.Time) ([]WorkspaceInstanceUsage, error) {
+type Order int
```

Comment:
I think this type doesn't need to be exported from the module, ie change to `order`.

Also, I'd move the type definition and consts up above the `func`s. Not sure if the style guide has anything to say about this. Feel free to ignore.

Reply:
It needs to be set by the api package when invoking it (at least it will need to when the order is specified in the request arguments)

Reply:
ok, so keep it exported because it will need to be for a later PR?

Reply:
You're correct that in this PR it doesn't need to be public, as it could move entirely into the ListUsage function. But because the direction is to expose in the `api` layer, it needs to be settable. 

But I think I just now understood what you mean, the type can be private, but the Consts can be public.

Reply:
> But I think I just now understood what you mean, the type can be private, but the Consts can be public.

💯

Reply:
Done.